src/build/__main__.py

```diff
@@ -96,6 +97,8 @@ def main_parser():  # type: () -> argparse.ArgumentParser
     """
     cwd = os.getcwd()
     out = os.path.join(cwd, 'dist')
+    # https://github.com/python/mypy/issues/1422
+    paths = _build.__path__  # type: Iterable[Optional[str]]  # type: ignore
```

Comment:
You'll need to add some details here. What makes ``__path__`` any better compared to ``__file__``? Also, why is this variable declared here if you're using ten lines below, do we really need a variable for it?

Reply:
`__path__` shows the path of the whole package, not just where `__init__.py` is. We are not a file package, we are a module package.

The variable is there to set the type. mypy does not recognize `module.__path__`, see the issue linked above.

Reply:
Isn't the path of the whole package ``os.path.dirname(build.__file__)``? I mean we did not yet split this package over multiple folders, so this feels to me like very premature optimization. One that requires a Github issue link, a type annotation, and slight confusion of how can one package has multiple paths. IMHO accommodating for a potential future where this can be in multiple places is a case of YAGNI.